### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Changelog for Thoth's Template GitHub Project
```
Comment (goern) [inline CHANGELOG.md]: this is not template

**Review by goern — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fridex) [reply CHANGELOG.md]: Fixed in 28daae4

**Review by pacospace — APPROVED** [review]
Amazing! Thanks!! :coffee:

**Comment by fridex** [comment]
Merging manually until zuul will be configured.

**Pull request merged**